OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -22,6 +22,14 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("Grant experience for team-kills.")]
 		public readonly bool FriendlyFire = false;
 
+		[Desc("Modifier (in percent) with which the `Experience` value will be multiplied",
+			"before being granted to an actor.")]
+		public readonly int ActorExperienceModifier = 10000;
```

Comment:
I know that this is a percentage. My question was if we really need this high value, `100 * 100`.
As we divide by [100](https://github.com/OpenRA/OpenRA/pull/11474/commits/70a58e82c2938222fb719bd5a44048e7d3119437#diff-73755936b9d513f9b2e7ad3a3f11bbd2R65) later anyway it wouldn't matter to just make this a "normal" number (no percentage), i.e. just `100`.